cscore/src/main/native/cpp/UsbCameraImpl.cpp

```diff
@@ -50,6 +50,8 @@ static constexpr char const* kPropWbValue = "white_balance_temperature";
 static constexpr char const* kPropExAuto = "exposure_auto";
 static constexpr char const* kPropExValue = "exposure_absolute";
 static constexpr char const* kPropBrValue = "brightness";
+static constexpr char const* kPropConnectVerbose = "connect_verbose";
+static constexpr unsigned kPropConnectVerboseId = 0;
```

Comment:
I assume `unsigned` is the argument of a system call elsewhere?

Reply:
It's used in the id of the property.  There's only one at present.